### ISSUE TITLE
NTR: L-glycerate high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8833]
New term request L-glycerate high in urine (elthat_100628030628):   UMLS: 4012187

**Comment by pnrobinson** [comment]
@MickeySegal
I am familiar with D-glyceric aciduria. What is the disease context of L-glycerate in the urine?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1283/ "In individuals with PH2, urinary L-glycerate and oxalate are generally elevated"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000670
- label: Elevated urinary L-glycerate level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: PMID:24116921
- def: The amount of L-glycerate in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: MPO-ANCA are host-derived auto-antibodies against shielded neutrophilic antigens. These antibodies react against primary granules present in neutrophils and monocytes. The formation of these antibodies has been hypothesized to be a two-step process. The first step involves exposure of neutrophils to inflammatory cytokines leading to surface exposure of cryptogenic antigens like myeloperoxidase or MPO. Next, predisposing genetic, environmental, and other factors result in the production of MPO-ANCA. In the second step, these MPO-ANCA cause damage to the host vasculature by reacting and crosslinking neutrophils to the endothelial receptors.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8833
- synonyms: MPO-ANCA(NONE)
